### ISSUE TITLE
Notebook fails to run at Parametric Halfspace inversion.

### ISSUE THREAD

**Issue by mariosgeo** [simpeg/segns-2024-tutorial#2]
Hi all,

To start with, thank you for this nice and useful tutorial. Unfortunately, I run to the following error, when I try to run the notebook, on 

     inv_prob_1 = inverse_problem.BaseInvProblem(
     phi_d_1, reg=empty_reg, opt=minimizer, beta=0.0
    )

    # sets up, phi_d + 0 * phi_m, meaning the minimizer is only going to act on the data misfit term.

Error message is

TypeError: Object must be an instance of BaseObjectiveFunction for dmisfit

Could you please advise

**Comment by jcapriot** [comment]
I would suspect your installed packages are not consistent with this repo’s [environment.yml](https://github.com/simpeg/segns-2024-tutorial/blob/1891d31fe92a5dd76f34d0700f478f0ddb0bfc14/environment.yml) file.

Double check against your installed versions.

**Comment by mariosgeo** [comment]
You are right, I suppose I used the yml from the main brunch to install the simpeg. 
I know it's out of scope, but is it possible to merge with the main brunch?
